utils/docker_test.go

```diff
@@ -553,6 +554,120 @@ func TestRunContainerWithArgs(t *testing.T) {
 
 }
 
+func TestRunContainer(t *testing.T) {
```

Comment:
The tests themselves look good within this test function, but would you mind breaking them out into their own test functions?

For example:
`TestRunContainerReturnErrorWhenSplitVolumeIs1`

Reply:
@everettraven 
I see, so we will have additional 4 test functions, instead of 1 test function with 4 subtests?

Reply:
> @everettraven 
> 
> I see, so by this PR we will have additional 4 test functions, instead of 1 test function with 4 subtests?

@neverbeenthisweeb Correct

Reply:
@everettraven 
Ok, will do the changes.